imap_processing/tests/lo/test_science_counts.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from imap_processing.lo.l0.data_classes.science_counts import ScienceCounts
+
+
+# Going to wait until validation data is available to test that
+# the values are correct. Currently only check for the shape
+# of the resulting data.
+@pytest.mark.skip(reason="no data to initialize with")
```

Comment:
Is it worth doing these shape checks for now even? I think that might be useful just to test setting up the classes and reading in your fake data. Just adding a TODO: comment indicating it isn't finished yet?

Reply:
I'm skipping these because the data class will crash without space_packet_parser packet output which I don't currently have. The unit tests will run if I comment out the init method for the data class though. So my plan was to comment that function out to test locally and skip the tests when I push to avoid GitHub action errors.

Reply:
xfail might be more appropriate here as well, but not a huge deal either way.
https://pytest.org/en/7.4.x/how-to/skipping.html
> An xfail means that you expect a test to fail for some reason. A common example is a test for a feature not yet implemented, or a bug not yet fixed. When a test passes despite being expected to fail (marked with pytest.mark.xfail), it’s an xpass and will be reported in the test summary.